### ISSUE TITLE
Nothing happens

### ISSUE THREAD

**Issue by gaziroffka** [boltgolt/howdy#759]
So. I've installed Howdy AUR from software center. Everything was smooth.
Configured everything. All was smooth aswell.
Tested. Added face. Smooth.
Added those recommended lines to /pam.d/
And I do not see a single blink of an emitter when I am asked for password. What can be wrong?
Just nothing happens.
Garuda linux / xfce de

**Comment by peterroth** [comment]
Did you configure Howdy with the `sudo howdy config` command?

**Comment by gaziroffka** [comment]
Yep. And all was good> Did you configure Howdy with the `sudo howdy config` command?

**Comment by peterroth** [comment]
Did you check if something is recorded in journalctl at the time when you try to use Howdy? Additionally, you can set Howdy (with the config file) to debug mode, so it will print out into a log file what's the problem.

**Comment by gaziroffka** [comment]
> 



> Did you check if something is recorded in journalctl at the time when you try to use Howdy? Additionally, you can set Howdy (with the config file) to debug mode, so it will print out into a log file what's the problem.

What do you mean by term 'using'? Configure is done and has no issues, and this is all i can do with Howdy, because nothing else happens!

**Comment by peterroth** [comment]
for example is you run `sudo howdy test`, does the window open and shows the picture of the cam? What does journalctl records when you run that command?

**Comment by gaziroffka** [comment]
> for example is you run `sudo howdy test`, does the window open and shows the picture of the cam? What does journalctl records when you run that command?

window does open and camera works just fine, I have added faces.
command:

sudo howdy test

Opening a window with a test feed

Press ctrl+C in this terminal to quit
Click on the image to enable or disable slow mode

QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-root'
QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-root'
^C
Closing window

I'm not so good with linux, and am not familiar with journalctl - what exactly i must command?

**Comment by gaziroffka** [comment]
> Great troubleshooting by @peterroth so far!
> 
> Where are you trying to activate Howdy? Have you tried using it with sudo?

I want Howdy to scan my face when password prompted. But Howdy does not activate, even though i added faces etc. Howdy works only to and add/test. That's all where it is active

**Comment by boltgolt** [comment]
Right, and if you run `sudo -i` you only get asked for a password and no other output

**Comment by gaziroffka** [comment]
> Right, and if you run `sudo -i` you only get asked for a password and no other output

exactly! 
sudoing, logging in, updating app, etc etc - Howdy does not start

**Comment by muhamed1ism** [comment]
I have same issue

**Comment by ipa1981** [comment]
In a new Ubuntu 23.04 I also facing the same. Everything set up, but no launching on sudo or login. Probably something related to PAM

**Comment by NickFlamel** [comment]
This worked for me: https://github.com/boltgolt/howdy/wiki/Howdy-on-Debian-Testing---KDE-Plasma#fix-the-script-for-debian-testing

**Comment by ipa1981** [comment]
> This fixed the issue for me: https://github.com/boltgolt/howdy/wiki/Howdy-on-Debian-Testing---KDE-Plasma#fix-the-script-for-debian-testing

This worked for me. Thanks @NickFlamel

**Comment by kiraitachi** [comment]
@NickFlamel  thanks mate that worked for me too on Kali Linux!

**Comment by AuraPy** [comment]
@NickFlamel Thank you SO much! Worked on Debian 12 LTS!
